src/base/logger.h

```diff
@@ -41,7 +41,7 @@ namespace Log
 {
     enum MsgType
     {
-        ALL = -1,
+        ALL = 0xF,
```

Comment:
Why?

Reply:
Seems c++20 requires signed number types to be two's complement, will revert.